tap_klaviyo/schemas/lists.json

```diff
@@ -16,11 +16,17 @@
     "name": {
       "type": "string"
     },
-    "type": {
```

Comment:
So this one was just named incorrectly?

Reply:
Yes it was named type but we are getting list_type in API response.